lib/validationtest.js

```diff
@@ -79,15 +82,20 @@ ValidationTest.prototype.groupRunTestData = async function(jobInfo, currentConte
 
 ValidationTest.prototype.initializeTestProduct2 = function(jobInfo, runTestData) {
     if (runTestData.dataPack && jobInfo) {
-        if (!this.testJobInfo.DefaultPriceList) {
-            this.testJobInfo.DefaultPriceList = jobInfo.DefaultPriceList;
+        if (!jobInfo.defaultPriceList) {
+            this.testJobInfo.errorHappen = true;
+            return VlocityUtils.error('Default PriceList Missing', 'VlocityRecordSourceKey', runTestData.dataPack.VlocityRecordSourceKey);
```

Comment:
This should not be 'VlocityRecordSourceKey' but: runTestData.dataPack.VlocityDataPackKey. The goal is point out where the error can be found.